### ISSUE TITLE
Move definition of number of lanes per sequencer into database from SequencingProcess

### ISSUE THREAD

**Issue by AmandaBirmingham** [jdereus/labman#266]
It appears that currently the number of lanes for each kind of sequencer is hard-coded into the labman.db.process.SequencingProcess class at 

https://github.com/jdereus/labman/blob/3cd014c1039a65542bce0f7e127d7db65699a66d/labman/db/process.py#L2408-L2410

I believe the kind names here are (a subset of the) values from the description field of the qiita.equipment_type table, which is populated in labman.db.support_files.db_patch_manual.sql, lines 39-45:

```
-- Populate the equipment type table
INSERT INTO qiita.equipment_type (description) VALUES
    ('echo'), ('mosquito'), ('EpMotion'), ('King Fisher'),
    ('tm 1000 8 channel pipette head'), ('tm 300 8 channel pipette head'),
    ('tm 50 8 channel pipette head'), ('HiSeq4000'), ('MiniSeq'), ('NextSeq'),
    ('HiSeq3000'), ('HiSeq2500'), ('HiSeq1500'), ('MiSeq'), ('NovaSeq'),
    ('Not applicable');
```

I worry that in the future, someone will add a new sequencer to the equipment table but forget to update the hardcoded lane values.

**Comment by josenavas** [comment]
Good point @AmandaBirmingham - the idea was to provide a GUI to add values to those tables (and ask for all the required information) but was low in priority since new sequencers is not something that happens pretty often :)

**Comment by charles-cowart** [comment]
As @AmandaBirmingham pointed out, a method for adding new sequencers should be present by full launch.

**Comment by antgonza** [comment]
Not sure this issue is an actual requirement for full launch, to put it in perspective: how often do we add a new sequencer and when do we expect to add a new one?

**Comment by unique-identifier** [comment]
@antgonza I feel like if and when it does happen, we'll totally forget to update this, if it's just buried in the code, but that is just my $0.02.

**Comment by AmandaBirmingham** [comment]
Yes, we could get away with going to full launch without this ... but we DO need to do it at some point.  I mean, we can put it off till the next milestone after full launch (what is that, by the way?) but we can't just dump it to "nice to have" bc as @charles-cowart points out, there is a strong chance we'll forget about it if it isn't standing in the way of achieving some milestone.

**Comment by AmandaBirmingham** [comment]
See https://github.com/jdereus/labman/issues/507#issue-442364742 re related issues to be fixed at same time as this.
